WordPress/build.gradle

```diff
@@ -91,6 +93,7 @@ android {
             buildConfigField "boolean", "ME_ACTIVITY_AVAILABLE", "false"
             buildConfigField "boolean", "TENOR_AVAILABLE", "false"
             buildConfigField "boolean", "READER_IMPROVEMENTS_PHASE_2", "false"
+            buildConfigField "long", "REMOTE_CONFIG_FETCH_INTERVAL", "3600"
```

Comment:
Honestly I haven't thought about this much, I've used the value from the tutorial but I think it would be fine to lower the time to something smaller (like 3 hours :-))